tests/infra/consortium.py

```diff
@@ -116,15 +116,14 @@ def generate_and_propose_new_member(self, remote_node, curve, member_data=None):
             new_member_id, curve, self.common_dir, self.share_script, self.key_generator
         )
 
-        proposal_body, careful_vote = self.make_proposal(
+        proposal_body, _ = self.make_proposal(
```

Comment:
Do we still use the second value returned by `make_proposal()` at all? If no, can we remove it?

Reply:
In infra no, but a client may well decide to use it.